deployment/document-understanding-solution.template

```diff
@@ -546,18 +575,23 @@ Resources:
                     },
                     {
                         "Effect": "Allow",
-                        "Resource": "arn:aws:cloudfront::${AWS::AccountId}:origin-access-identity/*",
+                        "Resource": "*",
```

Comment:
Could you elaborate on why you made this policy more permissive?

Reply:
after deleting the super open policy at the bottom (give "*" powers to "*" star resources), i actually had to add some extra permissions to the remaining policy blocks to account for the ones lost. This is one of those examples.

Reply:
I see.  Since the resources are all a part of Cloudfront, it it possible to reduce the scope to "arn:aws:cloudfront::${AWS::AccountId}:*"?
We had been given feedback from Solutions Builder Team to avoid * resources whenever possible

Reply:
i have tested that, and unfortunately it does not give enough permissions. Only "*" for resources gives enough permissions for it to deploy successfully.